pkg/repositories/gormimpl/description_entity_repo.go

```diff
@@ -0,0 +1,126 @@
+package gormimpl
+
+import (
+	"context"
+
+	"github.com/flyteorg/flyteadmin/pkg/common"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/flyteorg/flytestdlib/promutils"
+
+	flyteAdminDbErrors "github.com/flyteorg/flyteadmin/pkg/repositories/errors"
+	"github.com/flyteorg/flyteadmin/pkg/repositories/interfaces"
+	"github.com/flyteorg/flyteadmin/pkg/repositories/models"
+	"gorm.io/gorm"
+)
+
+// DescriptionEntityRepo Implementation of DescriptionEntityRepoInterface.
+type DescriptionEntityRepo struct {
+	db               *gorm.DB
+	errorTransformer flyteAdminDbErrors.ErrorTransformer
+	metrics          gormMetrics
+}
+
+func (r *DescriptionEntityRepo) Create(ctx context.Context, input models.DescriptionEntity) (uint, error) {
```

Comment:
who calls this since you always Create as a side effect of workflow and task registration?

Reply:
sorry, I forgot to remove this. removed it.